app/src/main/res/layout/item.xml

```diff
@@ -0,0 +1,57 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Use a more specific name than just `item`. Resource files are hard to manage if not named correctly.